src/teraranger_evo.cpp

```diff
@@ -0,0 +1,215 @@
+#include <ros/console.h>
+#include <string>
+
+#include <teraranger_hub/RangeArray.h>
```

Comment:
There was still an old reference to tr_hub_parser and there was a "generate_message_cpp" instead of "generate_messages_cpp"
This should fix the prerelease tool error